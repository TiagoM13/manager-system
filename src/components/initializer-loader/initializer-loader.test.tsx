import React from 'react';

import { render } from '@testing-library/react';

import { InitializerLoader } from '.';

describe('<InitializerLoader/>', () => {
  test('should render correctly component', () => {
    const { getByText, getByTestId, asFragment } = render(
      <InitializerLoader />,
    );

    const icon = getByTestId('icon-loader');

    expect(icon).toBeInTheDocument();
    expect(getByText('Carregando...')).toBeInTheDocument();

    expect(asFragment()).toMatchSnapshot();
  });
});
