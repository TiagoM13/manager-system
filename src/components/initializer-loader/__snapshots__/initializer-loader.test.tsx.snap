// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<InitializerLoader/> should render correctly component 1`] = `
<DocumentFragment>
  <div
    class="sc-braxZu dLKaNW"
  >
    <span
      class="loader"
      data-testid="icon-loader"
    />
    <span
      class="text-2xl text-sky-600 font-medium"
    >
      Carregando...
    </span>
  </div>
</DocumentFragment>
`;
